SYCL/SpecConstants/2020/handler-api.cpp

```diff
@@ -122,7 +122,7 @@ bool test_set_and_get_on_host(sycl::queue q) {
             "custom_type specializaiton constant before setting any value"))
       ++errors;
 
-    int new_int_value = 8;
+    int new_int_value = 0;
```

Comment:
I'd suggest to add new test cases rather than changing existing ones.

Reply:
I added new test case